### DISCUSSION TITLE
Let people choose multiple preferred languages

### DISCUSSION THREAD

**Discussion by avalonv** [community/community#39706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/29720696/203185826-4371c456-f2c5-4519-a276-efb5ff21c1d1.png)

I appreciate the "preferred spoken language" setting, as it allows me to highlight local projects I probably wouldn't find otherwise. However, being limited to just one language on seems very, well, limiting. 

This is a global platform. Millions of users speak English in addition to other languages, if not the majority.

I speak three languages, and I would like to see trending repositories for all of those languages, not just one. Also, I think the current paradigm heavily favours English, as it's the norm software development, so people are less likely to try other options if they're mutually exclusive (which makes them less likely to find new repos).

**Comment by ossanna16** [comment]
Hi there @avalonv and welcome to our community! Thank you for the feedback!
